### PR TITLE
Rename @eas/config to @expo/eas-json

### PR DESCRIPTION
**Pull request by fson**
We don't currently own the `@eas` org on npm, our npm org is named `@expo`. Additionally, I think the name `eas-json` will better communicate the decidedly narrow scope of this package ("A library for interacting with the eas.json") than the name `config`.

### CODE REVIEW

File: codecov.yml
Comment (dsokal) [inline codecov.yml:18]: Missing newline at EOF

**Review by dsokal — APPROVED** [review]
- I think this change is reasonable. 
- Please wait for @wkozyra95's review before merging.
- Fix lint warnings :)

**Review by wkozyra95 — APPROVED** [review]

**Pull request merged**
